metadata-service/war/src/main/webapp/WEB-INF/operationsServlet-servlet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this need to be its own entire servlet?

Reply:
Good point, I can move it under `openapi/`?

Reply:
Yep! Should have no issues with that